app/utils/SessionKeys.scala

```diff
@@ -67,6 +67,7 @@ object SessionKeys {
   val fileNames = "fileNames"
   val doYouWantToPayNow = "doYouWantToPayNow"
   val willUserPay = "willUsePay" //TODO: This should be set in the Can You Pay Your VAT Bill page
+  val canYouPay = "canYouPay"
```

Comment:
can this sessionKey be removed and use willUserPay instead? or remove the willUserPay key and just canYouPay? seems like we have two keys for the same thing

Reply:
I'll update